README.md

```diff
@@ -1,9 +1,11 @@
 Ember.js
 ========
 
-Shim repository for [Ember Application Framework](http://emberjs.com/).
+_Shim repository for [Ember Application Framework](http://emberjs.com/)._
 
 This package provides the core of the ember.js framework.
+When a new [build is released](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html),
+package managers will find it here.
```

Comment:
@rwjblue Just updated and rebased. I also added JSPM and Component to the list, as mentioned above.